superset/config.py

```diff
@@ -377,6 +377,18 @@ def _try_json_readsha(filepath: str, length: int) -> Optional[str]:
 # incomplete and not well maintained.
 LANGUAGES = {}
 
+
+# Override the default d3 locale format
+# Default values are equivalent to
+# D3_FORMAT = {
+#     "decimal": ".",           # - decimal place string (e.g., ".").
+#     "thousands": ",",         # - group separator string (e.g., ",").
+#     "grouping": [3],          # - array of group sizes (e.g., [3]), cycled as needed.
+#     "currency": ["$", ""]     # - currency prefix/suffix strings (e.g., ["$", ""])
+# }
+# https://github.com/d3/d3-format/blob/main/README.md#formatLocale
+D3_FORMAT: Dict[str, Any] = {}
```

Comment:
nit: as `TypedDict` is now a default thing in our currently supported Python versions, I'd prefer to create a `D3Format(TypedDict)` for this.